include/circt/Dialect/HW/HWAggregates.td

```diff
@@ -195,7 +195,7 @@ def StructExplodeOp : HWOp<"struct_explode", [NoSideEffect]> {
   let results = (outs Variadic<HWNonInOutType>:$result);
    let parser = "return ::parse$cppClass(parser, result);";
    let printer = "return ::print(p, *this);";
- }
+}
```

Comment:
Supreme nit: unrelated change. Try to keep this out in the future or slam it into `main` separately.